src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -130,6 +134,8 @@ public S3BlobStoreConfig getConfiguration() {
         }
     }
 
+    @SuppressFBWarnings(value = "MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR",
+            justification = "Calling load in the constructor is the typical way to manage GlobalConfiguration objects.")
```

Comment:
I guess as an alternative I could also make `S3BlobStoreConfig` implement `PersistentDescriptor`. Would you prefer that?

Reply:
No, https://github.com/jenkinsci/aws-global-configuration-plugin/blob/106dc1d8d86ee77da24c3bcf6f30bcf5959446e8/src/main/java/io/jenkins/plugins/aws/global_configuration/AbstractAwsGlobalConfiguration.java#L36 is correct as it stands.

The warning could be addressed just by making this class `final`.

Reply:
Oh, that's right. Thanks a lot!

I updated it.